README.md

```diff
@@ -72,94 +72,111 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
 Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on.
 
-### 0.61 - July 2022 Update
+### 0.62 - August 2022 Update
 
 This is a lighter release, with a shorter development cycle and focused on stability and improvements.
```

Comment:
62 is not a light release

Reply:
True

Reply:
Changing to `In this release, we focused on releasing three new PowerToys.`